.NET/Microsoft.Recognizers.Text.DataTypes.TimexExpression/English/TimexConvertEnglish.cs

```diff
@@ -95,16 +95,21 @@ public static string ConvertDate(TimexProperty timex)
                 return TimexConstantsEnglish.Days[timex.DayOfWeek.Value - 1];
             }
 
-            var month = TimexConstantsEnglish.Months[timex.Month.Value - 1];
```

Comment:
Could you please also move all hardcoded values to constants. Like "midday", "AM", etc, above.